src/encoded/docs/updating_ontologies.md

```diff
@@ -38,11 +38,11 @@ How to update the ontology versions
 
 6.  Update the following information
     
-    Site release version: 85
-    ontology.json file: ontology-2019-04-23.json
-    [UBERON release date]: 2018-10-14
-    [OBI release date]: 2018-08-27
-    [EFO release date]: 2019-04-18
+    Site release version: 88
```

Comment:
Should this be 89?

Reply:
Yes, thank you. I've fixed this in last push